app/build.gradle

```diff
@@ -109,16 +108,18 @@ dependencies {
     implementation "io.reactivex.rxjava2:rxjava:$rootProject.rxjavaVersion"
 
     //Dagger dependencies
-    annotationProcessor "com.google.dagger:dagger-compiler:$rootProject.daggerVersion"
+    kapt "com.google.dagger:dagger-compiler:$rootProject.daggerVersion"
     implementation "com.google.dagger:dagger:$rootProject.daggerVersion"
     compileOnly 'javax.annotation:jsr250-api:1.0'                //Required by Dagger2
 
     //Butter Knife
-    implementation "com.jakewharton:butterknife:$rootProject.butterKnifeVersion"
-    annotationProcessor "com.jakewharton:butterknife-compiler:$rootProject.butterKnifeVersion"
+    implementation "com.jakewharton:butterknife:$butterKnifeVersion"
+    kapt "com.jakewharton:butterknife-compiler:$butterKnifeVersion"
     implementation('com.crashlytics.sdk.android:crashlytics:2.6.7@aar') {
         transitive = true
     }
+    //Annotation library
+    implementation "androidx.annotation:annotation:1.1.0"
```

Comment:
Are we using this somewhere, please justify and if yes, prefer defining version constant and then using it, instead of directing using the version.

Reply:
The jetpack annotation library was needed in order for `kapt` to function properly without breaking the build. I will change the hardcoded version to use a variable shortly.